src/sql/src/plan/statement/ddl.rs

```diff
@@ -712,8 +712,21 @@ pub fn plan_create_view(
     } else {
         scx.allocate_name(normalize::object_name(name.to_owned())?)
     };
+    let (mut relation_expr, mut desc, finishing) =
+        query::plan_root_query(scx, query.clone(), QueryLifetime::Static)?;
+    relation_expr.bind_parameters(&params)?;
+    //TODO: materialize#724 - persist finishing information with the view?
+    relation_expr.finish(finishing);
+    let relation_expr = relation_expr.decorrelate();
     let replace = if *if_exists == IfExistsBehavior::Replace {
         if let Ok(item) = scx.catalog.resolve_item(&name.clone().into()) {
+            if relation_expr.global_uses().contains(&item.id()) {
+                bail!(
+                    "cannot replace {0}view {1}: depended upon by new {1} definition",
```

Comment:
useless, but are the positional references in the format string necessary here?

Reply:
(They allow you to refer to `item.name()` twice despite providing it as an argument just once.)

Reply:
Oh derp, I skipped over the `{1}` in the middle there!